libs/filameshio/src/MeshReader.cpp

```diff
@@ -159,30 +159,32 @@ static size_t fileSize(int fd) {
 
 namespace filamesh {
 
-MeshReader::Mesh MeshReader::loadMeshFromFile(filament::Engine* engine, const utils::Path& path,
-        MaterialRegistry& materials) {
-
+MeshReader::Mesh MeshReader::loadMeshFromFile(filament::Engine *const engine, const utils::Path& path,
+        MaterialRegistry& materialRegistry) {
     Mesh mesh;
 
-    int fd = open(path.c_str(), O_RDONLY);
+    /* Open the mesh file for reading only. */
```

Comment:
This comment isn't very useful

Reply:
OK. It's a choice.

Reply:
Not very useful for whom?

Reply:
The comment provides no useful information that can't be deducted from reading the code or from basic knowledge of common APIs. It's the equivalent of commenting `getAlpha()` with "Returns the alpha value.". Comments are great if they provide insights into what the code does and why (and to be fair, I'm sure we have many comments in Filament that aren't very useful).

Reply:
It is not the same thing.
For example, if a person has generally worked with more abstract functions like `fopen`, then what reason does one have to think that `open` will specifically open a file?